tyk-control-plane/values.yaml

```diff
@@ -228,10 +228,11 @@ global:
     # It is used to set TYK_MDCB_SECURITY_PRIVATECERTIFICATEENCODINGSECRET
     privateCertificateEncodingSecret: ""
 
-  # hashKeys specifies that if your Tyk Gateway is using hashed keys, set this value to true, so it matches.
-  # The Dashboard will now operate in a mode that is compatible with key hashing.
-  # It will enable hash keys for Tyk Gateway
-  # It is used to set TYK_GW_HASHKEYS
+  # hashKeys specifies that if your Tyk Gateway is using hashed keys, set this value to true so it matches.
+  # The Dashboard and MDCB will now operate in a mode that is compatible with key hashing.
+  # This takes precedence over the dashboard value i.e `dashboard.hashKeys`
```

Comment:
At the moment, `dashboard.hashKeys` takes precedence until we deprecate it. So, this comment needs to be updated accordingly. Or, we should update precedence order of hashKeys.

Reply:
looks good, thank you! also, as it is deprecated, we should mention it in the release notes and maybe remove the deprecated field in the future releases

Reply:
Thanks. Added this to https://github.com/TykTechnologies/tyk-docs/pull/4495 description as a reminder.